client/my-sites/site-settings/settings-performance/controller.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import PerformanceMain from 'my-sites/site-settings/settings-performance/main';
+
+export default {
```

Comment:
These days we try to avoid introducing `export default { func1, func2 }` for several reasons - in a nutshell to avoid unexpected issues when importing, and for build performance reasons.

Instead you could have just `export function performance() { ... }` or `export const performance = () => {}`

Reply:
Would it be something like this?

```
export function performance( context, next ) {
	context.primary = React.createElement( SiteSettingsPerformance );
	next();
}
```

Or do those have to be under a `return`?

Reply:
Yes, exactly - no need to be under a `return`. Here's an example just for reference: https://github.com/Automattic/wp-calypso/blob/a356b1e1ca3aba77360db20543d449da5c70a531/client/me/account-close/controller.js#L13

Reply:
Thanks for that! Done in https://github.com/Automattic/wp-calypso/pull/29597/commits/608cd72f13180d9f7985db5e811ff6bb65149c31